web/components/atoms/ProgressBar.tsx

```diff
@@ -5,7 +5,7 @@ interface Props {
 
 export default function ProgressBar({ progress }: Props) {
   return (
-    <div className="h-[6px] relative rounded-full bg-empty-gray">
+    <div className="h-[6px] w-[107px] lg:w-[250px] relative rounded-full bg-empty-gray">
```

Comment:
The progress bar looks SO GOOD especially the animation between pages :D The widths should be relative precents though for desktop vs mobile using `sm` instead of set sizes so that it adjusts better to different screen sizes